### ISSUE TITLE
Unable to Perform a BTRFS Send/Receive Using the Pipe "|" On Linux

### ISSUE THREAD

**Issue by x-radeon** [PowerShell/PowerShell#16055]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://github.com/PowerShell/PowerShell/releases)
- [X] Search the existing issues.
- [X] Refer to the [FAQ](https://github.com/PowerShell/PowerShell/blob/master/docs/FAQ.md).
- [X] Refer to [Differences between Windows PowerShell 5.1 and PowerShell](https://docs.microsoft.com/powershell/scripting/whats-new/differences-from-windows-powershell).

### Steps to reproduce

I am trying to write a PowerShell script to perform BTRFS snapshot management for Linux. The issue I'm running into is when I attempt to perform a BTRFS send and then pipe that output into a BTRFS receive, like the following:

`btrfs send -p /btrfs/send/snapshot/parent /btrfs/send/snapshot/new | btrfs receive /btrfs/receive/snapshot/`

It does not appear that PowerShell is handling the output correctly from `btrfs send`, because it throws a crc error when attempting to do this.

If I break the command into two separate commands, using a file, my send/receive works just fine (example below). So I know powershell correctly executes the `btrfs send` and `btrfs receive` commands, it's just something about how PowerShell is handling the piping between the commands.
`btrfs send -p $PreviousSnapshot $CurrentSnapshot -f /mnt/nvme/.snapshots/send_snap`
`btrfs receive -f /mnt/nvme/.snapshots/send_snap $DestinationDirectory`

### Expected behavior

```console
btrfs send -p /mnt/nvme/.snapshots/@GMT-2021.09.05-01.53.53/appdata /mnt/nvme/.snapshots/@GMT-2021.09.05-02.04.09/appdata | btrfs receive /mnt/hdd/nvme_snapshots/@GMT-2021.09.05-02.04.09/
At subvol /mnt/nvme/.snapshots/@GMT-2021.09.05-02.04.09/appdata
At snapshot appdata
```

### Actual behavior

```console
btrfs send -p -v /mnt/nvme/.snapshots/@GMT-2021.09.05-01.53.53/appdata /mnt/nvme/.snapshots/@GMT-2021.09.05-02.04.09/appdata | btrfs receive -v /mnt/hdd/nvme_snapshots/@GMT-2021.09.05-02.04.09/
At subvol /mnt/nvme/.snapshots/@GMT-2021.09.05-02.04.09/appdata
ERROR: crc32 mismatch in command
BTRFS_IOC_SEND returned 0
joining genl thread
```

### Environment data

```powershell
PS /mnt/nvme/.snapshots> $psversiontable

Name                           Value
----                           -----
PSVersion                      7.1.4
PSEdition                      Core
GitCommitId                    7.1.4
OS                             Linux 5.10.28-Unraid #1 SMP Wed Apr 7 08:23:18 PDT 2021
Platform                       Unix
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by rkeithhill** [comment]
Thanks for reporting this issue.  It is a duplicate of #1908,

**Comment by x-radeon** [comment]
Oh sorry, I only looked for "BTRFS" in the previous issues. I will look forward to this hopefully being resolved in 7.3!

Thanks!
